tests/test_parsers.py

```diff
@@ -2,27 +2,31 @@
 Tests for ralph.parsers module.
 """
 import gzip
-import logging
 import shutil
 
-import pandas as pd
 import pytest
 
-from ralph import filters
 from ralph.parsers import BaseParser, GELFParser
 
 
 def test_baseparser_parse():
     """Test that calling parse on the BaseParser raises a NotImplementedError."""
+    # pylint: disable=abstract-class-instantiated
 
-    with pytest.raises(NotImplementedError):
+    with pytest.raises(
+        TypeError,
+        match="Can't instantiate abstract class BaseParser with abstract methods parse",
+    ):
         BaseParser().parse("foo.log")
 
     # pylint: disable=abstract-method
     class MyParser(BaseParser):
         """Dummy parser."""
 
-    with pytest.raises(NotImplementedError):
+    with pytest.raises(
```

Comment:
Same here, you can remove this test. All of this is in the python engine, you can I think trust it.